lib/shared/layers/python-sdk/python/genai_core/models.py

```diff
@@ -48,6 +52,22 @@ def list_openai_models():
         if model["id"].startswith("gpt")
     ]
 
+def list_azure_openai_models():
+    # azure openai model are listed, comma separated in AZURE_OPENAI_MODELS variable in external API secret
+    models = genai_core.parameters.get_external_api_key("AZURE_OPENAI_MODELS")
+
+    return [
+        {
+            "provider": Provider.AZURE_OPENAI.value,
```

Comment:
It seems this PR is missing the update of `Provider` enum to include `AZURE_OPENAI`